airflow/providers/docker/operators/docker_swarm.py

```diff
@@ -93,20 +95,37 @@ class DockerSwarmOperator(DockerOperator):
         Supported only if the Docker engine is using json-file or journald logging drivers.
         The `tty` parameter should be set to use this with Python applications.
     :type enable_logging: bool
+    :param configs: List of ConfigReferences that will be exposed to the service
+    :type configs: list
+       Example: [{'source': my-conf, 'target': where/config/should/be/.env}]
+    :param secrets: List of SecretReference to be made available inside the containers
+    :type secrets: list
+    :param networks: List of network names or IDs to attach the service to
+    :type networks: list
     """
 
     @apply_defaults
     def __init__(
             self,
             image,
             enable_logging=True,
+            networks=None,
+            configs=None,
+            secrets=None,
             *args,
             **kwargs):
         super().__init__(image=image, *args, **kwargs)
 
         self.enable_logging = enable_logging
+        self.networks = networks
+        self.configs = configs
+        self.secrets = secrets
+        self.service_name = self._get_service_name()
```

Comment:
We should not initialize this field in the constructor because the constructor is called each time the DAG is loaded.